### ISSUE TITLE
No database update since upgrade to v24.2.18

### ISSUE THREAD

**Issue by inkly** [RaBa64/OpenDTU-Database#22]
### What happened?

I have pushed version 24.2.18 this morning; there have been no database updates since.

### To Reproduce Bug

upgrade to v24.2.18

### Expected Behavior

production by hour on graph and api :-)

### Install Method

Pre-Compiled binary from GitHub

### What git-hash/version of OpenDTU?

v24.2.18

**Comment by nonflammable** [comment]
me too
json file ends before upgrade

**Comment by nonflammable** [comment]
I think I've found a problem. After the update, the DTU module's communication frequency changes automatically, and data is no longer received.

You need to adjust the frequency using the slider to the one required in your country. For the EU, the default should be 865MHz, but I might be wrong.

Settings -> DTU Settings

![Zrzut ekranu 2024-02-20 111214](https://github.com/RaBa64/OpenDTU-Database/assets/6041490/a47859ab-76a8-4e2b-81f9-50929bfc431b)

**Comment by andy-kuebke** [comment]
Same here

**Comment by MartinNK** [comment]
Works fine for me Upgrade from original opendtu version withour datatbase to prerelease database. Upload database.bin ... made a manual reboot via webinterface.

my next try is to append 2 databasefiles ... because of reset since 3. jan ... no data with same date in both files. Do database file have a header or only the datasets?

**Comment by andy-kuebke** [comment]
> Maybe we need a reboot after database.bin upload?

Easy test! Just restarted mine via Webinterface and let it run for now... we'll see tomorrow.

**Comment by andy-kuebke** [comment]
No, just restarting via web interface didn't do the trick (as expected).

**Comment by nonflammable** [comment]
24.2.24 - in my case the update solved the problem.

**Comment by inkly** [comment]
Same here, thanks :)
